docs/src/dev/io/gryo.asciidoc

```diff
@@ -160,6 +160,8 @@ As of TinkerPop 3.3.0, there is now a new version of Gryo in 3.0 that is only pa
 to use 3.0 serializers with 1.0 serializers will likely lead to failure. For best results, users should always
 utilize the same version of TinkerPop on the client as on the server.
 
+IMPORTANT: As of 3.6.0, Gryo `MessageSerializer` implementations have been removed from the codebase.
```

Comment:
Shouldn't we also at least remove this sentence from line 127:

> This characteristic makes it ideal for use in Gremlin Server, which is designed to return arbitrary results of varying types

as Gryo can now only be used a file I/O format?

Reply:
agreed - removed the sentence.